scripts/borg.android.spec

```diff
@@ -0,0 +1,46 @@
+# -*- mode: python -*-
+# this pyinstaller spec file is used to build borg binaries on posix platforms
+
+import os, sys
+
+#basepath = '/vagrant/borg/borg'
+basepath = '/home/borg/borg/borg'
```

Comment:
looks like this is specific to your developer system?
so you will change this to the vagrant path before it gets merged?